src/components/BShuttle/BShuttle.vue

```diff
@@ -0,0 +1,404 @@
+<template>
+  <div class='shuttle'>
+
+    <div class='available'>
+      <label id='available-label'>{{ availableLabel }}</label>
+      <ul
+        aria-labeledby='available-label'
+        role='selectbox'
+      >
+        <li
+          v-if="placeholder && (!options || options.length === 0)"
+          :aria-label="placeholder"
+          class='placeholder'
```

Comment:
This or the `ul` needs `tabindex="0"`. I'll test it after standup to figure out which one.

Reply:
i don't think we want the placeholder to ever be selected, so I'm going to but it on the ul

Reply:
The only reason I would have chosen the placeholder over the `ul` is that we don't want the `ul` to have focus when there are options so the tabindex would need to be based on whether the options are shown or not.